.github/drafts/pull_request_template.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Pull Request Template
+about: Add a new icon or feature to the repo.
+title: 'New Icon/Feature: [NAME]'
+assignees: ''
+
+---
+# New Icon Section
+If you are adding a new icon, delete the **New Feature** section down below and follow the steps listed here.
+
+## Double check these details before you open a PR**
+*Tick the checkboxes to ensure you've done everything*
+- [] PR does not match another non-stale PR currently opened
+- [] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)* as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
+- [] Your icons are put in a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
+- [] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
+- [] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)
+
+Refer to the [`CONTRIBUTING.md`](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#contributing-to-devicon) for more details.
+
+
+## Link to the official page to prove your SVG is correct and up to date.
+*Link goes here*
```

Comment:
This will make it a comment and only visible when making the PR or editing the comment.
```suggestion
<!-- Link goes here -->
```